hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -0,0 +1,11 @@
+module HookContextExtension
```

Comment:
Using the same name as in the Katello hook feels fragile. Probably best to rename the one in Katello to `KatelloHookContextExtension`.

Reply:
This was marked as resolved, but in `foreman-installer/katello/hooks/boot/01-kafo-hook-extensions.rb` it still uses the same name.